docs/js-core-1/week-2/lesson.md

```diff
@@ -429,6 +430,53 @@ So to sum up, `for of` loops are pretty similar to `for` loops, but:
 
 Next week, we will think more about the most common iterable object in JavaScript: [arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
 
+## PD (Dealing with conflict in teams)
+
+Adapted from [Tectonica Interpersonal Conflict Resolution](https://github.com/Techtonica/curriculum/blob/main/career/conflict-resolution.md).
+
+Whenever people spend more than a little time together, there might be conflict. You will learn ways to work through these situations while staying respectful and honest.
+
+In this session you will talk about
+- what conflict is and the common ways we address it
+- how can we add to our conflict resolution toolkits and become better at solving conflict effectively
+- what effective body language is
+- what effective choice of words is
+- how to build trust among teams
+
+[Conflict resolution slides](https://docs.google.com/presentation/d/1-3Dwx9iImzwWvhfX2Qd_zoGb5Q1jP1_bgf7gZ-ZzzjE/edit#slide=id.g1b73cb08c45_0_6)
+
+Notes for the facilitator: Remember to adapt/update the content in line with your cohort’s needs and arrange timings as needed.
+
+**Exercise 1 (15 minutes)**
```

Comment:
To format the exercises the same way as the rest of the syllabus, please use the :::note markers.

Reply:
I tried to user those markers, it didn't work. I'll try again.